src/electron.ts

```diff
@@ -95,35 +96,19 @@ app.on('activate', () => {
 });
 
 function registerShortcuts(win: Electron.BrowserWindow): void {
-  globalShortcut.register('CommandOrControl+T', () => win.webContents.send('newTab', true));
-  globalShortcut.register('CommandOrControl+N', () => createWindow());
-  globalShortcut.register('CommandOrControl+Shift+Left', () => win.webContents.send('tabLeft', true));
-  globalShortcut.register('CommandOrControl+Shift+Right', () => win.webContents.send('tabRight', true));
-  globalShortcut.register('CommandOrControl+1', () => win.webContents.send('switchTab', 0));
-  globalShortcut.register('CommandOrControl+2', () => win.webContents.send('switchTab', 1));
-  globalShortcut.register('CommandOrControl+3', () => win.webContents.send('switchTab', 2));
-  globalShortcut.register('CommandOrControl+4', () => win.webContents.send('switchTab', 3));
-  globalShortcut.register('CommandOrControl+5', () => win.webContents.send('switchTab', 4));
-  globalShortcut.register('CommandOrControl+6', () => win.webContents.send('switchTab', 5));
-  globalShortcut.register('CommandOrControl+7', () => win.webContents.send('switchTab', 6));
-  globalShortcut.register('CommandOrControl+8', () => win.webContents.send('switchTab', 7));
-  globalShortcut.register('CommandOrControl+9', () => win.webContents.send('switchTab', 8));
-  globalShortcut.register('CommandOrControl+0', () => win.webContents.send('switchTab', 9));
+  let keypressFunctions = {
+    'send': function(key, value){ win.webContents.send(key, value); },
+    'toggleDevTools': function(){ win.webContents.toggleDevTools(); },
+    'createWindow': function(){ createWindow(); }
+  };
+
+  keyboardShortcuts.forEach( shortcut => {
+    globalShortcut.register(shortcut.keypress, () => keypressFunctions[shortcut.sctype](shortcut.sckey, shortcut.scvalue));
```

Comment:
`globalShortcut` is a part of electron package https://electron.atom.io/docs/api/global-shortcut/, I don't think renaming this would make much sense, what do you think?